backend/db/utils.py

```diff
@@ -182,6 +182,9 @@ def refresh_derived_tables_exec(
 # todo: move this somewhere else, possibly load.py or db_refresh.py
 # todo: what to do if this process fails? any way to roll back? should we?
 # todo: currently has no way of passing 'local' down to db status var funcs
+# todo: last_derived_refresh_request and last_derived_refresh_exited: these should ideally be schema specific. That way,
```

Comment:
Minor optimization todo: allow `test_n3c` and `n3c` refreshes to run simultaneously.